server/Procfile

```diff
@@ -1 +1 @@
-web: node index.js
\ No newline at end of file
+web: npx sequelize-cli db:migrate && node index.js
```

Comment:
Would  we need env variable `--env production`
Running locally, this was the command - `heroku run npx sequelize-cli db:migrate --env production --app toilet-paper-app`

Reply:
i was able to run this through heroku run. unclear how it picks production by default though

```
➜  covid-maps git:(master) heroku run npx sequelize-cli db:migrate
Running npx sequelize-cli db:migrate on ⬢ toilet-paper-app... up, run.4457 (Hobby)

Sequelize CLI [Node: 12.16.1, CLI: 5.5.1, ORM: 5.21.6]

Loaded configuration file "config/config.json".
Using environment "production".
No migrations were executed, database schema was already up to date.
```

Reply:
Great